src/lamplib/src/genny/tasks/run_tests.py

```diff
@@ -163,10 +163,14 @@ def out(cmd_output: str) -> bool:
 # See the logic in _setup_resmoke.
 # These are the "Binaries" evergreen artifact URLs for mongodb-mongo compile tasks.
 # The binaries must be compatible with the version of the mongo repo checked out in use for resmoke,
-# which is the sha "cf8ddbdf99ae7fb2d96ca9f350224ffa193e40b3" mentioned below.
+# which is the sha "6e2dcc5a39eb2de9d2e8209271115c078ae16470" mentioned below.
 _canned_artifacts = {
-    "osx": "https://mciuploads.s3.amazonaws.com/mongodb-mongo-master/macos/cf8ddbdf99ae7fb2d96ca9f350224ffa193e40b3/binaries/mongo-mongodb_mongo_master_macos_cf8ddbdf99ae7fb2d96ca9f350224ffa193e40b3_21_12_03_09_47_48.tgz",
-    "amazon2": "https://mciuploads.s3.amazonaws.com/mongodb-mongo-master/amazon/cf8ddbdf99ae7fb2d96ca9f350224ffa193e40b3/binaries/mongo-mongodb_mongo_master_amazon_cf8ddbdf99ae7fb2d96ca9f350224ffa193e40b3_21_12_03_09_47_48.tgz",
+    "osx": "https://mciuploads.s3.amazonaws.com/mongodb-mongo-master/macos/6e2dcc5a39eb2de9d2e8209271115c078ae16470/binaries/mongo-mongodb_mongo_master_macos_6e2dcc5a39eb2de9d2e8209271115c078ae16470_21_12_03_09_47_48.tgz",
```

Comment:
I see that osx is in the mciuploads bucket, is that a straight-up evg build artifact? If so, should we put it in dsi-donot-remove as well?

Reply:
Also, can we document either in a comment here or in one of the .md files how to update these canned artifacts if needed?